src/EnergyPlus/CurveManager.cc

```diff
@@ -2127,7 +2125,7 @@ namespace Curve {
                     format("GetCurveInput: Currently exactly one (\"1\") {} object per simulation is required when using the AirflowNetwork model.",
                            CurrentModuleObject));
                 ErrorsFound = true;
-            } else if (numOfCPArray == 1) {
+            } else {
```

Comment:
[src/EnergyPlus/CurveManager.cc:2130]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 2124.

Reply:
Pretty smart there, CppCheck.